scripts/bin_to_c.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
```

Comment:
Please also run `pep8 bin_to_c.py` and resolve the issues reported.

Reply:
ok, thanks.